win32ss/user/winsrv/usersrv/register.c

```diff
@@ -45,6 +49,30 @@ CSR_API(SrvRegisterLogonProcess)
         LogonProcessId = 0;
     }
 
+#if 1 //HAAAACK. This should be done in UserClientConnect which is never called!
+
```

Comment:
Please add a `DPRINT1` with `HAAAAACK` and a CORE-issue describing the problem :)

Reply:
This would take years to fix and will just add noise to the log.

Reply:
If it is only called once or twice at boot, then it should be fine, and make it more visible that this lacks.
If it's printed at every process start, it is too much noise.

Reply:
The ones who can remotely fix the problem are aware of it, it existed for years already. If I add such a print we will probably disable it again in one or two years.